pkg/pgmodel/migrations/sql/idempotent/base.sql

```diff
@@ -1525,15 +1552,38 @@ CREATE OR REPLACE VIEW SCHEMA_INFO.metric AS
      *
     FROM SCHEMA_CATALOG.metric_view();
 
+DROP VIEW IF EXISTS SCHEMA_INFO.label;
```

Comment:
The drop view shouldn't go into the idempotent file. Rather it has to go into a migration file. Take a look at https://github.com/timescale/promscale/pull/291/files#diff-3a439eaeefc867aa848294a901f6931fa90f20ed559f28c3c8db20c3d1ada565 for an example of a migration file. Note that this requires bumping the version (https://github.com/timescale/promscale/pull/291/files#diff-73b8ec8a05213857da6e9598e34c5d7a18a6e5349250908ae6710578236977f2R36)